services/app-web/src/common-code/dateHelpers/calendarDate.ts

```diff
@@ -1,19 +1,35 @@
+import { CalendarDate } from '../healthPlanFormDataType'
 import { dayjs } from './dayjs'
 
 // We store calendar dates (dates with no time info) in UTC for consistency.
 // This formats a date correctly like '12/02/2022' for display.
-function formatCalendarDate(date: Date | undefined): string {
+function formatCalendarDate(date: CalendarDate | undefined): string {
+    if (!date) {
+        return ''
+    }
+    const parsedCalendarDate = date.split('-')
+    const month = parsedCalendarDate[1]
+    const day = parsedCalendarDate[2]
+    const year = parsedCalendarDate[0]
+    return `${month}/${day}/${year}`
+}
+
+function formatDateTime(date: Date | undefined): string {
```

Comment:
Are we still using this function? We probably don’t want the `tz.(‘UTC’)` for this if so since we should probably just resepct the timezone the Date was set in.

Reply:
still using, and fixed